File: Baekjoon/1924.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
	int x, y;
	int month[12] = { 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
	string days[7] = {"SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"};
	int idx = 0;
	cin >> x >> y;

	for (int i = 0; i<x; i++) {
		if ((i + 1) == x)
			idx += y;
		else
			idx += month[i];
	}	
	
	cout << days[idx % 7] << '\n';

	return 0;
}

File: Baekjoon/1924.kt
fun main() {
    var (x, y) = readLine()!!.split(' ').map { it.toInt() }
    var cur = 1
    val m = listOf(31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31)
    val d = listOf("SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT")
    while (cur < x) {
        y += m[cur - 1]
        cur++
    }
    println(d[y % 7])
}